authors/apps/profiles/urls.py

```diff
@@ -1,11 +1,24 @@
 from django.urls import path
 
 from authors.apps.profiles.views import ProfileRetrieveAPIView
+from authors.apps.profiles.views import FollowUserRetrieveAPIView
+from authors.apps.profiles.views import FollowersListAPIView
+from authors.apps.profiles.views import UserFollowingListAPIView
+
 
 app_name = "profiles"
 
 urlpatterns = [
     # url to view the profile of a user, does not require permissions
     path('<str:username>', ProfileRetrieveAPIView.as_view(),
          name='view_profile'),
+    # Url to follow a user
```

Comment:
I think the comments are not necessary given that the names of the endpoints are self-explanatory and obvious.